persistence-modules/spring-jpa-3/pom.xml

```diff
@@ -1,48 +1,52 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>spring-jpa-3</artifactId>
-    <version>0.1-SNAPSHOT</version>
-    <name>spring-jpa-3</name>
-    <packaging>war</packaging>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>spring-jpa-3</artifactId>
+	<version>0.1-SNAPSHOT</version>
+	<name>spring-jpa-3</name>
+	<packaging>war</packaging>
```

Comment:
We need to revert the spacing to use spaces, not tabs. There should not be this many changes to the pom.xml.